server/src/main/assets/stylesheets/main.less

```diff
@@ -307,12 +307,12 @@ a:hover {
   color: #a94442;
 }
 
-.multi-day-export .modal-content, .arrivals-export .modal-content {
+.multi-day-export .modal-content, .arrivals-export .deployments-export .modal-content {
```

Comment:
We can rename the existing `arrivals-export` class to be more generic, eg `terminal-export` to make the same component usable for arrivals, deployments and recommendations.